xcodeproj/xctarget_test.go

```diff
@@ -0,0 +1,139 @@
+package xcodeproj
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/bitrise-io/go-utils/fileutil"
+	"github.com/bitrise-io/go-utils/pathutil"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_GivenOnlyApp_WhenProjectTargetCalled_ThenExpectSingleTarget(t *testing.T) {
```

Comment:
NOTE: This test naming is quite new, in other places is usually just Test_ProjectTarget() fro example. It could also be a table driven testm where the name of the test could contain the same infos (GivenOnlyApp_WhenProjectTargetCalled_ThenExpectSingleTarget)

Reply:
Fair point and I have wanted to do it that way but I've felt that the `expectedTargets` are too big to fit into that scheme and this way would be more readable.